autotx/agents/SendTokensAgent.py

```diff
@@ -46,8 +47,26 @@
         "token": "UNI"
     }}
 
+    Example 3:
+    User: Send 10 USDC to 0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045 then buy 5 UNI with ETH and send 40 WBTC to 0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045
+    Call prepare_transfer_transaction with args:
+    {{
+        "amount": 10,
+        "receiver": "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045",
+        "token": "USDC"
+    }}
+    NOTE: the second transfer was not prepared because it's waiting for the swap transaction to be prepared first.
```

Comment:
the second transfer has nothing to do with the asked swap, right? is that intended?

Reply:
Yep, this is just to illustrate to the LLM that the transactions should be created in order as defined by user